app/src/main/res/raw/repositories.json

```diff
@@ -109,5 +109,6 @@
   {"name":"TWRP", "url":"https://gerrit.twrp.me/"},
   {"name":"Unlegacy Android", "url":"https://gerrit.unlegacy-android.org/"},
   {"name":"UsFirst Collabnet", "url":"https://usfirst.collab.net/gerrit/", "ci_accounts": "^(Jenkins.*)$"},
-  {"name":"Wireshark", "url":"https://code.wireshark.org/review/", "ci_accounts": "^(.* Buildbot)$"}
+  {"name":"Wireshark", "url":"https://code.wireshark.org/review/", "ci_accounts": "^(.* Buildbot)$"},
+  {"name":"XenonHD", "url":"https://gerrit.xenonhd.com"}
```

Comment:
Hi Miguel,

Thanks for the PR.

Build failed (https://travis-ci.org/jruesga/rview/builds/481738478) because of ModelTestHelper test. Repositories urls must end with a slash (https://github.com/jruesga/rview/blob/master/app/src/test/java/com/ruesga/rview/misc/ModelHelperTest.java#L90).

Please, could you fix it before merge it? Thx.

Regards

Reply:
Hey mate, Ofc I can really sorry